lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1670,6 +1679,41 @@ Error ONNXModelLoader::loadTile(const ONNX_NAMESPACE::NodeProto &op,
   return Error::success();
 }
 
+Error ONNXModelLoader::loadExpand(const ONNX_NAMESPACE::NodeProto &op,
+                                  const ArgumentDictionaryTy &dict) {
+  const std::string &opName = loadOperatorName(op);
+  NodeValue in;
+  Constant *repeats;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
+  ASSIGN_VALUE_OR_RETURN_ERR(repeats, getConstantByName(op.input(1)));
+
+  std::vector<int64_t> tiles;
+  helperSetter<int64_t, int64_t>(repeats, tiles);
+  auto inputDimSize = (size_t)in.dims().size();
+  auto repeatSize = (size_t)tiles.size();
+  if (repeatSize > inputDimSize) {
+    for (size_t i = 0; i < repeatSize - inputDimSize; i++) {
```

Comment:
nit: We follow LLVM style for `for` loops even when not using an iterator ([see here](https://llvm.org/docs/CodingStandards.html#don-t-evaluate-end-every-time-through-a-loop)):

`for (size_t i = 0, e = repeatSize - inputDimSize; i < e; i++)`

Reply:
Done.